.gitignore

```diff
@@ -0,0 +1,3 @@
+composer.lock
+vendor
+phpunit.xml
```

Comment:
I am leaving this to be excluded because for testing we'll be putting our own local environment variables inside of it, for instance, the app url will go there instead of making a ".env" file because when testing, you aren't using the package inside of a full laravel project.

This is why in my TestCase.php file inside of the tests folder I'm overriding the setUp method to not fully setup a Laravel project the way you would if you installed this package inside of your overall Laravel project.

In further tests, I'll be adding the setting up of Testbench for mocking aspects of Laravel like testing database calls and such.